src/HealthChecks.Publisher.ApplicationInsights/ApplicationInsightsPublisher.cs

```diff
@@ -23,8 +23,9 @@ class ApplicationInsightsPublisher
         private static readonly object sync_root = new object();
         private readonly bool _saveDetailedReport;
         private readonly bool _excludeHealthyReports;
+        private readonly bool _trackExceptions;
 
-        public ApplicationInsightsPublisher(string instrumentationKey = default, bool saveDetailedReport = false, bool excludeHealthyReports = false)
+        public ApplicationInsightsPublisher(string instrumentationKey = default, bool saveDetailedReport = false, bool excludeHealthyReports = false, bool trackExceptions = true)

```

Comment:
How you can change TrackException parameters? probably you need to add a new overload or default  parameter on  ApplicationInsightsHealthCheckBuilderExtensions.cs

Reply:
You are correct, I was going to add a default on `ApplicationInsightsHealthCheckBuilderExtensions`. Again, the plan is adding a default `bool trackExceptions = true` rather than false because I think having a report on exceptions happened is something one would like for default (at least I know it's what I wanted).

Reply:
I've add the parameter to the extension method in 0c2e3894765a1c0bbcef39fb7f6da4c996f00410.